generator/templates/generator/seed.html

```diff
@@ -10,7 +10,7 @@
     <div class="container">
       <div class="pt-2 pb-2">
         <h1>Download Seed</h1>
-        <h3 class="pb-2"><a target="_blank" href="{% url 'generator:share' share_id %}">Seed share link</a></h3>
+	<h3 class="pb-2"><a id="seed_share" target="_blank" href="{% url 'generator:share' share_id %}">Seed share link</a> &middot;&middot;&middot; <a id="seed_practice" target="_blank" href="{% url 'generator:practice' share_id %}">(create practice seed)</a></h3>
```

Comment:
oh right, I added IDs here. they aren't important right now but I was playing with having a nice button to copy the share link to the clipboard and having an ID on the `a` would be helpful there, at least.